docs/core/distribution-packaging.md

```diff
@@ -166,6 +166,18 @@ The following lists the recommended packages:
   - **Version:** \<sdk version>
   - **Contains:** (15)
 
+The following 2 meta packages are optional. They do bring value for end users, as they abstract the top-level package (dotnet-sdk) which simplifies the installation of the full set of .NET packages. These meta packages would reference a specific .NET SDK version.
+
+- `dotnet[major]` - Installs the specific sdk version
+  - **Version:** \<sdk version>
+  - **Example:** dotnet7
+  - **Dependencies:** `dotnet-sdk-[major].[minor]`
+
+- `dotnet` - Installs the specific sdk version determined by distros to be the primary, usually the latest available
```

Comment:
```suggestion
- `dotnet` - Installs a specific SDK version determined by distros to be the primary version&mdash;usually the latest available
```

Reply:
Yes, they should be updated to SDK - thanks.

Reply:
Thanks! I'll commit your suggestions and make the other change.